### ISSUE TITLE
Build failing due to inability to find ctapipe

### ISSUE THREAD

**Issue by aribrill** [cta-observatory/dl1-data-handler#60]
The Travis build is currently failing in master on the following step:

```
$ pip install .
Processing /home/travis/build/cta-observatory/dl1-data-handler
Collecting numpy>=1.15.0 (from dl1-data-handler==0.7.5)
  Downloading https://files.pythonhosted.org/packages/c1/e2/4db8df8f6cddc98e7d7c537245ef2f4e41a1ed17bf0c3177ab3cc6beac7f/numpy-1.16.3-cp36-cp36m-manylinux1_x86_64.whl (17.3MB)
Collecting tables>=3.4.4 (from dl1-data-handler==0.7.5)
  Downloading https://files.pythonhosted.org/packages/ab/79/4e1301a87f3b7f27aa6c9cb1aeba4875ff3edb62a6fe3872dc8f04983db4/tables-3.5.1-cp36-cp36m-manylinux1_x86_64.whl (4.3MB)
Collecting ctapipe>=0.6.2 (from dl1-data-handler==0.7.5)
  ERROR: Could not find a version that satisfies the requirement ctapipe>=0.6.2 (from dl1-data-handler==0.7.5) (from versions: none)
ERROR: No matching distribution found for ctapipe>=0.6.2 (from dl1-data-handler==0.7.5)
The command "pip install ." failed and exited with 1 during .
```

The problem seems to be that pip cannot find ctapipe, which makes sense as it's not available on PyPI. Previously, when the build was succeeding, we were installing ctapipe directly from GitHub ([last successful commit](https://github.com/cta-observatory/dl1-data-handler/blob/abb04abf989d7bafb7c5179c80ed62bf522088c6/setup.py)).

I see at least two approaches. We could revert to installing directly from GitHub. Alternatively, we could switch to using conda for installation, so we can use ctapipe's recommended installation method using conda. A concern is that the most recent ctapipe conda package may not be compatible with the version of eventio we need (see #52). Any opinions or other ideas?

**Comment by vuillaut** [comment]
Hi.
What changed that makes the build to fail?

**Comment by aribrill** [comment]
@vuillaut what changed was that the links to install a particular dependency version directly from GitHub were removed from setup.py. At least, I think that's what the cause was.

**Comment by vuillaut** [comment]
right I see that
`ctapipe @ https://api.github.com/repos/cta-observatory/ctapipe/tarball/v0.6.2'`
was removed from `setup.py` by @bryankim96 in commit 75d966deb3842c117e73c0ea78bcfdd906aa79af
But I don't understand the reason behind this removal 🤔

**Comment by vuillaut** [comment]
I would recommend moving entirely to conda.
For a quick fix, see #61.

**Comment by nietootein** [comment]
Hi @vuillaut. @bryankim96 was working towards a new DL1DH conda package that would fix #52, but that effort was paused waiting for ctapipe 0.7 to be out. That may be the reason behind that removal. The plan is, as you suggest, to generate conda packages for every future release, that will be uploaded to cta-observatory conda channel.

**Comment by vuillaut** [comment]
Thanks @nietootein 
It was indeed my understanding when I looked again at the last commits.
So the answer is basically waiting that @bryankim96 finishes his work towards full conda?

**Comment by nietootein** [comment]
My suggestion would be to go for your quick fix for the time being. As far a I remember, we can not go full conda right now because ctapipe 6.2.0 conda package comes with the requirement of eventio==0.11, and eventio will need to be upgraded to 0.16.1 via pip after a conda installation of dl1-data-handler... Thoughts? If you guys agree I think we should approve #61 and postpone the conda installation for after the summer conferences.

**Comment by aribrill** [comment]
I agree with the plan to implement the quick fix from @vuillaut now and postpone the switch to full conda assuming that there would be merge conflicts if we did that now. @bryankim96 has been looking into this so it would be good to hear from him first.

**Comment by vuillaut** [comment]
> My suggestion would be to go for your quick fix for the time being. As far a I remember, we can not go full conda right now because ctapipe 6.2.0 conda package comes with the requirement of eventio==0.11, and eventio will need to be upgraded to 0.16.1 via pip after a conda installation of dl1-data-handler... Thoughts? If you guys agree I think we should approve #61 and postpone the conda installation for after the summer conferences.

I don't think that's a problem.
We can have an `environement.yml` for the setup of the env. which will install ctapipe via conda (including eventio 0.11) and then will install eventio 0.16.1 via pip or conda.

**Comment by MaxNoe** [comment]
As conda uses a sat solver, it will say that the environment is not achievable.

**Comment by bryankim96** [comment]
I didn't have any time to work on this yesterday evening. My take is that given that vuillaut already made a PR restoring the previous installation method I also support using it (at least until the next ctapipe conda release). 

Regarding finishing moving over completely to conda, I had forgotten about the issue Daniel in #52; I asked Karl a little while ago about this and it seems like although we can fix it by re-releasing v0.6.2 w/ changed dependencies, we will need him to do this for us and it may take a while for him to find time to do it. The easiest thing is probably to wait for v0.6.3 (or v0.7.0) as everyone suggests. 

Regarding installing eventio v0.16.0 with pip in an environment.yml file, I think that would probably work, but I have been shying away from using environment files (in favor of a single conda package w/ dependencies) because I've had some issues with reliably installing across systems w/ environment files.
